### PR TITLE
	modified:   src/AzSK.ADO/Framework/Configurations/SVT/ADO/ADO.Projec…

### PR DESCRIPTION
**Pull request by t-rbavdekar**
…t.json

	modified:   src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Project.ps1

## Description
</br>
 Brief description of the requested change.

## Checklist
- [ ] I have read the instructions mentioned in [Contribute to Code](/Contributing.md).
- [ ] I have read and understood the criteria described under [submitting changes](/Contributing.md#submitting-changes). 
- [ ] The title of the PR clearly describes the intent of the PR.
- [ ] This PR does not introduce any breaking changes to the code.

### CODE REVIEW

**Comment by t-rbavdekar** [comment]
Code Review for ADO_Project_AuthZ_Dont_Grant_All_Pipelines_Access_To_Secure_Files

File: src/AzSK.ADO/Framework/Configurations/SVT/ADO/ADO.Project.json [hunk 1]
```diff
@@ -250,7 +250,7 @@
       "ControlSeverity": "High",
       "Automated": "Yes",
       "MethodName": "CheckSecureFilesPermission",
-      "Rationale": "To support security of the pipeline operations, secure files must not be granted access to all pipelines. This is in keeping with the principle of least privilege because a vulnerability in components used by one pipeline can be leveraged by an attacker to attack other pipelines having access to critical resources.",
+      "Rationale": "Secure files must not be granted access to all pipelines. An unauthorized user can steal information from the secure files by building a pipeline and accessing the secure file",
       "Recommendation": "To remediate this, go to Project -> Pipelines -> Library -> Secure Files -> select your secure file from the list -> click Security -> Under 'Pipeline Permissions', remove pipelines that secure file no more requires access to or click 'Restrict Permission' to avoid granting access to all pipelines.",
```
Comment (Garima-msft) [inline src/AzSK.ADO/Framework/Configurations/SVT/ADO/ADO.Project.json:254]: Please update the recommendation in this format:

Steps to remediate: 1. Navigate to the build pipeline using resource link -> 2. Triggers -> 3. Pull request validation -> 4. Forks -> 5. Uncheck 'Make secrets available to builds of forks'.",

Comment (Garima-msft) [inline src/AzSK.ADO/Framework/Configurations/SVT/ADO/ADO.Project.json:254]: PR should be raised against develop branch

File: src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Project.ps1 [hunk 2]
```diff
@@ -687,16 +687,18 @@ class Project: ADOSVTBase
...
+            $projectId = ($this.ResourceContext.ResourceId -split "project/")[-1].Split('/')[0]
+            $url = "https://dev.azure.com/$($this.OrganizationContext.OrganizationName)/$($projectId)/_apis/distributedtask/securefiles?api-version=6.1-preview.1"
             $response = [WebRequestHelper]::InvokeGetWebRequest($url);
             # check on response object, if null -> no secure files present
             if(([Helpers]::CheckMember($response[0],"count",$false)) -and ($response[0].count -eq 0)) {
                 $controlResult.AddMessage([VerificationResult]::Passed, "There are no secure files present.");
             }
             # else there are secure files present
-            elseif((-not ([Helpers]::CheckMember($response[0],"count"))) -and ($response.Count -gt 0)) {
+            #WHY elseif and not else
```
Comment (Garima-msft) [inline src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Project.ps1:699]: Is this something you want to discuss? If yes, do not check in this comment in code

Also test all the scenarios of control eval to see nothing breaks due to change from else if to else

**Pull request closed without merging**